docs/docs/legal/vulnerability-disclosure-policy.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Vulnerability Disclosure Policy
+custom_edit_url: null
+--- 
+
+## Introduction
+
+At ZITADEL we are extremely grateful for security aware people who disclose vulnerabilities to us and the open source community.
+All reports will be investigated by our team and we will work with you closely to validate and fix vulnerabilities reported to us.
+
+We require that you keep vulnerabilities confidential until we are able to address them, since public disclosure of security vulnerabilities could put the ZITADEL community at risk.
+
+ZITADEL (CAOS Ltd.) will not take legal action against you or terminate your access to our services, conditional that you report vulnerabilities in accordance to this policy.
+
+## Scope
+
+The scope of this policy applies to all Websites and Services operated by ZITADEL.
+
+All security issues that concern our Product in form of Software in our [open source repositories](https://github.com/zitadel), should be reported according to [Security Policy](https://github.com/zitadel/zitadel/blob/main/SECURITY.md).
+
+When in doubt about the scope of your vulnerability, please follow the process outlined in this policy.
+
+## Discovering a vulnerability
+
+Responsible security research on our Websites, Products, and Services is encouraged and we allow you to conduct testing on our services to which you have authorized access.
+
+You must not do research or testing that involves
+
+- Any activity that violates applicable law
+- Modify or destroy any data that does not belong to you
+- Accessing or attempt to access data that does not belong to you
+- Executing denial of service attacks
+- Executing load testing
```

Comment:
```suggestion
- Executing load testing without prior consent from ZITADEL
```

Reply:
I have that in the next sentence after this list. Because exceptions could be granted in special cases for a few of them (eg, DoS)

Reply:
In case of load testing it felt to me that the sentence does not apply since there might not be a report.
We can go with your formulation and reconsider later if needed.

Reply:
Ah ok. Yes, let's do that. We've also mentioned load testing anyways on the rate limit / fair use policy. Hope that should not lead to any issue for now.